modules/ROOT/pages/editor-important-options.adoc

```diff
@@ -55,5 +54,7 @@ include::partial$configuration/referrer_policy.adoc[leveloffset=+1]
 
 include::partial$configuration/suffix.adoc[leveloffset=+1]
 
-\{% comment %} Not documented, but probably belongs in a new section here somewhere.
-include::partial$configuration/iframe_attrs.adoc[] \{% endcomment %}
+////
```

Comment:
noted to followup

Reply:
Tyler left a number of these throughout the docs. It's something we can look at post 6.0 as they weren't documented in 5.x either.